docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+Add a Custom Flow
+^^^^^^^^^^^^^^^^^
+
+To define a new flow for your project, simply add the name of the new flow under the ``flows`` section of your ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    greet_and_sleep:
+        group: projectName
+        description: Greets the user and then sleeps for 5 seconds.
+        steps:
+            1:
+                task: command
+                options:
+                    command: echo 'Hello there!' 
+            2:
+                task: util_sleep
+
+This is a flow comprised of two tasks: ``command`` greets the user by echoing a string, and ``util_sleep`` then tells CumulusCI to sleep for five seconds. You can reference how the flows are defined in the `universal cumulusci.yml <https://github.com/SFDO-Tooling/CumulusCI/blob/master/cumulusci/cumulusci.yml>`_ file.
+
+
+Add a Flow Step
+^^^^^^^^^^^^^^^
+
+If you want to add a step to a flow, use ``cci flow info <name>`` first to see the existing steps.
+
+    Example: ``dev_org``
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+There are four steps to this flow. The first three steps are themselves flows, and the last step is a task.
+
+All *non-negative numbers and decimals* are valid as step numbers in a flow.
+You can add steps before, in-between, or after existing flow steps.
+If you want to add a step at the beginning of the dev org flow, valid step numbers would include any number greater than or equal to zero and less than 1. Example values would include 0, 0.3, and 0.89334. All of these would cause the step to execute before step 1 in the ``dev_org`` flow.
+
+If you want to add a step *between* steps 2 and 3, then a step number greater than 2 or less than 3 can be used.
+
+If you want to add a step *after* all steps in the flow, then any step number greater than 4 can be used.
```

Comment:
These sentences are all related to the idea of inserting steps. Consider making them unordered list items under "You can add steps before..." instead of discrete paragraphs. The relationships would be clearer, and more readable as a result.